package.json

```diff
@@ -12,7 +12,8 @@
     "@colbyfayock/semantic-release-pnpm": "^1.2.2",
     "@semantic-release/changelog": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
-    "semantic-release": "^23.0.0"
+    "semantic-release": "^23.0.0",
+    "typescript": "^5.5.4"
```

Comment:
I find it a good practice to keep typescript as the root's dev dep. It makes it possible to select the installed TS version in VS Code through "Select TypeScript Version...". If you don't do that VS Code sticks to its pre-installed TS version and that can lead to weird discrepancies between the IDE and `tsc`

Reply:
I dont recall now what are the exact rules behind pnpm's lookup for executables. You could try removing it from packages and try executing `pnpm run tsc` from a package to see what happens. If it succeeds then it's likely OK to remove `typescript` dependency from ur packages. I usually keep it defined everywhere but it might be mostly out of habit ;p